conda_pypi/downloader.py

```diff
@@ -59,6 +59,15 @@ def find_and_fetch(finder: PackageFinder, target: Path, package: str):
     link = result.best and result.best.link
     if not link:
         raise CondaPypiError(f"No PyPI link for {package}")
+
+    # Check if the file is a wheel (.whl)
     filename = link.url_without_fragment.rsplit("/", 1)[-1]
+    if not filename.endswith(".whl"):
+        raise CondaPypiError(
```

Comment:
It looks like the `result` object above [also has an `applicable` and `candidate` ](https://unearth.readthedocs.io/en/latest/api/finder.html#unearth.finder.BestMatch). Should this also check if there are wheel files in these results?